src/main/java/org/dcsa/bkg/model/transferobjects/BookingTO.java

```diff
@@ -0,0 +1,110 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import org.dcsa.core.events.model.enums.*;
+import org.dcsa.core.validator.ValidVesselIMONumber;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@Data
+public class BookingTO {
+
+  private String carrierBookingRequestReference;
+
+  @NotNull(message = "ReceiptDeliveryTypeAtOrigin is required.")
+  private ReceiptDeliveryType receiptDeliveryTypeAtOrigin;
+
+  @NotNull(message = "DeliveryTypeAtDestination is required.")
+  private ReceiptDeliveryType deliveryTypeAtDestination;
+
+  @NotNull(message = "CargoMovementTypeAtOrigin is required.")
+  private CargoMovementType cargoMovementTypeAtOrigin;
+
+  @NotNull(message = "CargoMovementTypeAtDestination is required.")
+  private CargoMovementType cargoMovementTypeAtDestination;
+
+  @NotBlank(message = "ServiceContractReference is required.")
+  @Size(max = 30, message = "ServiceContractReference has a max size of 30.")
+  private String serviceContractReference;
+
+  private PaymentTerm paymentTerm;
+
+  @NotNull(message = "CargoGrossWeightUnit is required.")
+  private CargoGrossWeight cargoGrossWeightUnit;
```

Comment:
This has been removed in the IM 3.5 document (might have been updated since I saw it last week).